tbdex/offering/offering.go

```diff
@@ -49,27 +49,27 @@ type PayoutDetails struct {
 
 // PayinMethod represents a single payment option on an Offering.
 type PayinMethod struct {
-	Kind                   string `json:"kind"`
-	Name                   string `json:"name,omitempty"`
-	Description            string `json:"description,omitempty"`
-	Group                  string `json:"group,omitempty"`
-	RequiredPaymentDetails string `json:"requiredPaymentDetails,omitempty"` // TODO: change to JSON Schema type
-	Fee                    string `json:"fee,omitempty"`
-	Min                    string `json:"min,omitempty"`
-	Max                    string `json:"max,omitempty"`
+	Kind                   string          `json:"kind"`
+	Name                   string          `json:"name,omitempty"`
+	Description            string          `json:"description,omitempty"`
+	Group                  string          `json:"group,omitempty"`
+	RequiredPaymentDetails json.RawMessage `json:"requiredPaymentDetails,omitempty"` // TODO: change to JSON Schema type
```

Comment:
is this type a string? or a raw json map

Reply:
great q @jiyoontbd . it's technically bytes under the hood. it's basically a type that can be used to tell `json.Marshal` to take exactly what's there instead of attempting to marshal